Components/LineData.gd

```diff
@@ -113,6 +115,19 @@ func _parse_line() -> void:
 	detected_macro = _get_macro_script()
 
 
+func _count_tabs() -> int:
+	var tabs_count = 0
+
+	for i in source_text.length():
```

Comment:
Funfact:

For c: String in text:

will iterate over the individual characters!